bin/start-shuffle-server.sh

```diff
@@ -93,6 +100,13 @@ JVM_ARGS=" -server \
           -XX:+PrintAdaptiveSizePolicy \
           -XX:+PrintGCDateStamps \
           -XX:+PrintGCTimeStamps \
+          -XX:+CrashOnOutOfMemoryError \
+          -XX:+ExitOnOutOfMemoryError \
+          -XX:+PrintTenuringDistribution \
```

Comment:
I'm not sure if these parameters will affect performance. PTAL @zuston @jerqi

Reply:
We use this args in PRD, so PTAL.

Reply:
I'm ok.

Reply:
+1